Formula/openssl@1.1.rb

```diff
@@ -23,14 +23,9 @@ class OpensslAT11 < Formula
 
   keg_only :shadowed_by_macos, "macOS provides LibreSSL"
 
-  on_linux do
-    resource "cacert" do
-      # homepage "http://curl.haxx.se/docs/caextract.html"
-      url "https://curl.haxx.se/ca/cacert-2020-01-01.pem"
-      mirror "https://gist.githubusercontent.com/dawidd6/16d94180a019f31fd31bc679365387bc/raw/ef02c78b9d6427585d756528964d18a2b9e318f7/cacert-2020-01-01.pem"
-      sha256 "adf770dfd574a0d6026bfaa270cb6879b063957177a991d453ff1d302c02081f"
-    end
+  depends_on "ca-certificates"
```

Comment:
Let's bump the `revision` here. Will we need to stop setting `HOMEBREW_FORCE_BREWED_CURL` on Mojave for things to work here?

Reply:
> Will we need to stop setting `HOMEBREW_FORCE_BREWED_CURL` on Mojave for things to work here?

Yes.

Was hoping to bump the revision at the same time as making it keg-only, but that's going to be a bit of work since OpenSSL dependent testing is busted right now.